firestore-counter/POSTINSTALL.md

```diff
@@ -20,13 +20,13 @@ match /databases/{database}/documents/pages/{page} {
 
 #### Set up a scheduled function
 
-Review the [scheduled function documentation](https://firebase.google.com/docs/functions/schedule-functions) to set up a call to `${function:controller.url}` every minute. You may need to enable some APIs in your Firebase project to use scheduled functions.
+Review the [scheduled function documentation](https://firebase.google.com/docs/functions/schedule-functions) to set up a call to `${function:aggregator.url}` every minute. You may need to enable some APIs in your Firebase project to use scheduled functions.
```

Comment:
Revise to:

Review the [scheduled function documentation](https://firebase.google.com/docs/functions/schedule-functions) to send a message to the PubSub topic `${param:EXT_INSTANCE_ID}` every minute.